packages/react-resize-handle/src/hooks/useResizeHandle.ts

```diff
@@ -154,6 +159,33 @@ export const useResizeHandle = (params: UseResizeHandleParams) => {
         const newValue = clampWithMode(value, minValue, maxValue, relative);
 
         if (newValue !== currentValue.current) {
+          if (!relative) {
```

Comment:
can we hoist the relative logic and cleanup here to make that code path less relevant like

```ts
if (relative) {
  return relativeHandlingDeprecated(...args);
}

// stuff that works
```

Reply:
also would be nice to have a GH issue for this

Reply:
Makes sense, will do 👍

Reply:
Diff became a bit ackward